src/ray/core_worker/core_worker.h

```diff
@@ -186,6 +187,9 @@ struct CoreWorkerOptions {
   int runtime_env_hash;
   /// The PID of the process for setup worker runtime env.
   pid_t worker_shim_pid;
+  /// The startup token of the process assgined to it
+  /// during startup via command line arguments.
```

Comment:
```suggestion
  /// during startup via command line arguments. This is needed because the actual core worker process may not have the same pid as the process the worker pool starts (due to shim processes). 
```

Reply:
Should I change this everywhere or doing it only once here would suffice?